source/Calamari.Shared/Integration/Packages/Download/DockerImagePackageDownloader.cs

```diff
@@ -60,6 +63,14 @@ public PackagePhysicalFileMetadata DownloadPackage(string packageId,
                                                            int maxDownloadAttempts,
                                                            TimeSpan downloadAttemptBackoff)
         {
+            if (variables.Get(AuthenticationVariables.FeedType) == FeedType.AwsElasticContainerRegistry.ToString())
```

Comment:
this is not a request for a change, but when we do this for ACR and GCR, this could be refactored to individual methods, for each type.